common/src/main/java/com/wynntils/models/character/type/SavableSkillPointSet.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © Wynntils 2023-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.character.type;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public record SavableSkillPointSet(
+        int strength,
+        int dexterity,
+        int intelligence,
+        int defence,
+        int agility,
+        List<String> armourNames,
+        List<String> accessoryNames) {
+    /**
+     * Constructs a new SavableSkillPointSet representing just a loadout.
+     */
+    public SavableSkillPointSet(int[] skillPoints) {
+        this(skillPoints, new ArrayList<>(), new ArrayList<>());
```

Comment:
Should be List.of() so it is an unmodifiable list

Reply:
Doesn't work with gson for some reason, when I try to save anything with List.of() it does not persist after relaunch

Reply:
Yeah, that might be the case. I'll just make sure to support it separately in a new PR eventually. Just use the immutable wrappers here for now, and i'll handle proper serialization support later.